modules/ingest-common/src/test/java/org/elasticsearch/ingest/common/IngestRestartIT.java

```diff
@@ -95,7 +96,9 @@ public void testFailureInConditionalProcessor() {
         );
         assertTrue(e.getMessage().contains("this script always fails"));
 
-        NodesStatsResponse r = client().admin().cluster().prepareNodesStats(internalCluster().getNodeNames()).setIngest(true).get();
+        NodesStatsResponse r = client().admin().cluster().prepareNodesStats(internalCluster().getNodeNames())
+            .addMetric(NodesStatsRequest.Metric.INGEST.metricName())
```

Comment:
Not that it would save a lot of bytes, but what do you think about adding an override that takes a `NodesStatsRequest.Metric`?

```suggestion
            .addMetric(NodesStatsRequest.Metric.INGEST)
```

This would be a bit more type-safe at least?

Reply:
I'd be tempted use some static imports to abbreviate the code a little, too.

Reply:
The `NodesStatsRequest.Metric` construct is just temporary and will eventually be replaced with a more pluggable solution. That is to say, the `addMetric` method will need to accept strings that are defined by plugins. Those strings will have to be validated dynamically, possibly by the `NodeService`.

Reply:
I agree on static imports improving readability; I'll make that change.